src/lib/evil/evil_time.h

```diff
@@ -23,29 +23,6 @@
  * @{
  */
 
```

Comment:
```suggestion
EAPI int gettimeofday(struct timeval * tp, struct timezone * tzp);
```

Reply:
Ok, I will do that

Reply:
I think something happened here, the `gettimeofday` definition is gone.

Reply:
don't forget to sorround it with `_MSC_VER` like you did in .c file.